HealthGPT/HealthGPT/HealthDataInterpreter.swift

```diff
@@ -0,0 +1,51 @@
+//
+// This source file is part of the Stanford HealthGPT project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University & Project Contributors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Spezi
+import SpeziFHIR
+import SpeziOpenAI
+import Foundation
+
+class HealthDataInterpreter<ComponentStandard: Standard>: DefaultInitializable, Component, ObservableObject, ObservableObjectProvider {
+    required init() {}
```

Comment:
By moving the chat as a property inside the `HealthDataInterpreter`, you can then use a `didSet` observer of the property to do the similar functionality as defined in the `getAnswer` function that is currently found in the view.